recipe/meta.yaml

```diff
@@ -7,8 +7,8 @@ package:
   version: {{ version }}
 
 source:
-  fn: graphviz-{{ version }}.zip
-  url: https://files.pythonhosted.org/packages/source/g/graphviz/graphviz-{{ version }}.zip
```

Comment:
If the URLs work reliably, then we can use `files.pythonhosted.org`. However, for a long time that wasn't the case (even after that linked comment was posted). Hence why `pypi.io` is used throughout conda-forge and recommended in our [example]( https://github.com/conda-forge/staged-recipes/blob/f73e2912d7b7ad5814d11b49e26b6044a3cf8525/recipes/example/meta.yaml#L18 ).